Observer/AdyenCcDataAssignObserver.php

```diff
@@ -53,7 +58,12 @@ class AdyenCcDataAssignObserver extends AbstractDataAssignObserver
         self::ENCRYPTED_EXPIRY_MONTH,
         self::ENCRYPTED_EXPIRY_YEAR,
         self::HOLDER_NAME,
-        self::VARIANT
+        self::VARIANT,
+        self::JAVA_ENABLED,
```

Comment:
We can use an object to add all browser related information similar to given in standard helper functions.
https://github.com/Adyen/adyen-3ds2-js-utils (refer BrowserInfo)

Reply:
This one is for validating the parameters passed by the frontend in the additional_data array. In this case we follow the Magento 2 approach while creating these constants.